twilight-cache-inmemory/Cargo.toml

```diff
@@ -1,18 +1,16 @@
 [package]
-authors = ["Twilight Contributors"]
+authors.workspace = true
 categories = ["caching"]
 description = "In-process-memory based cache for the Twilight ecosystem."
-documentation = "https://docs.rs/twilight-cache-inmemory"
```

Comment:
I am not sure if there is a general way missing documentation keys are handled, e.g. if it is only crates.io that will use docs.rs automatically so I think we should leave them for now since they are not really a part of the MSRV bump. unless of course there is a reason to include it here and similar for the other comment.